examples/MEGRE_multiecho/dataset_description.json

```diff
@@ -5,5 +5,5 @@
         "Kirstie Whitaker",
         "Gilles de Hollander"
     ],
-    "Description": "This virtual dataset is collected using a single FLASH sequence, which yields multiple images, corresponding to different echo times. This data has been used to calculate a quantitative susceptibility map, as well as a T2* map."
+    "Description": "This virtual dataset is collected using a single MEGRE sequence, which yields multiple images, corresponding to different echo times. This data has been used to calculate a quantitative susceptibility map, as well as a T2* map."
```

Comment:
This is a great change ✅ 

This is what I was thinking of when I said add the information to the `dataset_description.json` file - adding the text you have describing the dataset in the BEP001_examples.md file to the json itself 😄